WavesCS/Transactions/UnknownTransaction.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using DictionaryObject = System.Collections.Generic.Dictionary<string, object>;
+
+namespace WavesCS
+{
+    public class UnknownTransaction : Transaction
+    {
+        int Type;
+
+        public UnknownTransaction(byte[] senderPublicKey, int type) : base(senderPublicKey)
+        {
+            Type = type;
+        }
+
+        public UnknownTransaction(Dictionary<string, object> tx) : base(tx)
+        {
+            Type = tx.GetInt("type");
+        }
+
+        public override byte[] GetBody()
+        {
+            using (var stream = new MemoryStream())
```

Comment:
No need to implement these methods (both GetBody and GetJson), I guess. We can just throw exception.